centos-ci/README.md

```diff
@@ -0,0 +1,37 @@
+# LeApp CI
+
+LeApp CI is hosted on the [CentOS CI infrastructure](http://ci.centos.org/) using Jenkins and since
+our tests are virtualization heavy we have a dedicated slave node (slave03) for our tests.
+
+## Architecture
+
+When Jenkins is triggered via a webhook it uses a Duffy key located in `~/duffy.key` to get a node from the CICO
+cluster on which the test will run. Duffy is CentOS CI middleware for resource management inside the cluster.
+The lifetime (read, debuggability) of a failed test run is 12 hours.
```

Comment:
I'll ask on the CI mailing list, perhaps the timeout can be tweaked via the JJB file, it's good to have this documented. I think it would be better to stick with 12 hours and adopt good logging hygiene like you mention - after all, with longer timeout we might end up blocking *more* bare metal nodes and I'm afraid the CI guys wouldn't like us for that :)